File: php/1929-concatenation-of-array.php
class Solution {

    /**
    * @param Integer[] $nums
    * @return Integer[]
    */
    function getConcatenation($nums) {
        $n = count($nums);
        $ans = array_fill(0, $n*2, 0);
        for ($i = 0; $i < $n; $i++) {
            $ans[$i] = $nums[$i];
            $ans[$i + $n] = $nums[$i];
        }
        return $ans;
    }
}

File: golang/1929-concatenation-of-array.go
package golang

func getConcatenation(nums []int) []int {
    n := len(nums)
    ans := make([]int, n*2)
    for i := 0; i < n; i++ {
        ans[i] = nums[i]
        ans[i + n] = nums[i]
    }
    return ans
}